assets/js/logic.js

```diff
@@ -1 +1,134 @@
-// empty for init
+// localStorage
+
+function allBooks() {
+  let books;
+  if (localStorage.getItem('books') === null) {
+    books = [];
+  } else {
+    books = JSON.parse(localStorage.getItem('books'));
+  }
+  return books;
+}
+
+function addBooks(newBook) {
+  const getAllBooks = allBooks();
+  getAllBooks.push(newBook);
+  localStorage.setItem('books', JSON.stringify(getAllBooks));
+}
+
+function deleteBooks(title, author) {
+  const books = allBooks();
+
+  books.forEach((book, index) => {
+    if (book.title === title && book.author === author) {
+      books.splice(index, 1);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use array methods such as `slice` over `splice`, that doesn't mutate the array.